### DISCUSSION TITLE
add more clef

### DISCUSSION THREAD

**Discussion by sugizo** [CoderLine/alphaTab#448] [Ideas]
# Description 

please add more clef 
**ref** 
https://en.wikipedia.org/wiki/Clef

# Possible Solutions

the clef not available on alphatab : 
- French violin
- Baritone
- Sub-bass
- Mezzo-soprano
- Soprano

# Current Workarounds 

vexflow already support this clef

thanks

**Comment by Danielku15** [comment]
Thanks for your request. Can you also provide the details on which file formats you would expect it to work? The clefs supported today are the ones supported by Guitar Pro. MusicXML might support some more.

**Comment by sugizo** [comment]
both format will be nice to have
not use Guitar Pro, ATM and never use MusicXML
will try both ASAP and give further suggestion about this 
thanks
